### PR TITLE
Standardize error messages for commands

### LINKED ISSUE #112: Standardise invalid command error messages

### PR DESCRIPTION
**Pull request by thony-ui**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java [hunk 1]
```diff
@@ -22,11 +22,9 @@ public class DeleteMeetingCommand extends Command {
...
             + ": Deletes the meeting identified by its title and date.\n"
             + "Parameters: "
-            + PREFIX_MEETING_TITLE + " "
-            + PREFIX_MEETING_DATE + "\n"
-            + "Example: " + COMMAND_WORD + " "
-            + PREFIX_MEETING_TITLE + "Meeting 1 "
-            + PREFIX_MEETING_DATE + "01-01-2024";
+            + PREFIX_MEETING_TITLE + "TITLE "
+            + PREFIX_MEETING_DATE + "DATE" + "\n"
+            + "Restrictions: " + "t/Title has to be a valid String and d/Date has to be in the format xx-xx-xxxx";
```
Comment (yongkheehou) [inline src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java]: could you specify the format as `dd-mm-yyyy` instead?

**Comment by Virusrwj223** [comment]
Looks ok. You may want to consider using String.format rather than string concatanation.

File: src/main/java/seedu/address/logic/commands/AddBuyerCommand.java [hunk 2]
```diff
@@ -20,9 +20,11 @@ public class AddBuyerCommand extends AddClientCommand {
      * Describes the format and usage of the addbuyer command.
      * Includes the command word and expected parameters inherited from {@link AddClientCommand#CLIENT_PARAMETERS}.
      */
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a buyer to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a buyer to the address book. \n"
             + "Parameters: "
-            + AddClientCommand.CLIENT_PARAMETERS;
+            + AddClientCommand.CLIENT_PARAMETERS + "\n"
+            + AddClientCommand.CLIENT_RESTRICTIONS;
+
```
Comment (Virusrwj223) [inline src/main/java/seedu/address/logic/commands/AddBuyerCommand.java]: May consider String.format() here and the rest of the changes below.

File: src/main/java/seedu/address/logic/commands/AddClientCommand.java [hunk 3]
```diff
@@ -22,6 +22,8 @@ public abstract class AddClientCommand extends Command {
     public static final String CLIENT_PARAMETERS = PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL";
+    public static final String CLIENT_RESTRICTIONS = "Restrictions: "
```
Comment (yongkheehou) [inline src/main/java/seedu/address/logic/commands/AddClientCommand.java]: why do we skip the email and name restrictions though? if we want to specify restrictions, should we specify all of them?

File: src/main/java/seedu/address/logic/commands/DeleteBuyerCommand.java [hunk 4]
```diff
@@ -16,11 +16,13 @@
...
+    public static final String PARAMETER_PHONE = String.format("%sPHONE", PREFIX_PHONE);
     public static final String MESSAGE_ARGUMENTS = "phoneNumber: %1$s";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the buyer identified by the phone number used in the displayed person list.\n"
-            + "Parameters: phone number (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " " + PREFIX_PHONE + "81621234";
+            + ": Deletes a buyer from the address book.\n"
+            + "Parameters: " + PARAMETER_PHONE + "\n"
+            + "Restrictions: " + PARAMETER_PHONE
+            + " is an 8 digit positive integer which starts with the number 8 or 9";
```
Comment (yongkheehou) [inline src/main/java/seedu/address/logic/commands/DeleteBuyerCommand.java]: should we create a `DeleteClientCommand` so we can create the CLIENT_RESTRICTIONS static constant like how u did it for the `AddClientCommand`? There's a little bit of code duplication between the `DeleteBuyerCommand` and `DeleteSellerCommand`

**Review by apollo-tan — APPROVED** [review]
LGTM!

**Pull request merged**